packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -100,6 +93,7 @@ export class Database {
       return {
         ...data,
         _collection: collection.name,
+        _keepTemplateKey: !!collection.templates,
```

Comment:
Indicates whether the `bridge` should keep the `_template` property when persisting. This was previously handled elsewhere but is now a higher-level concern